npm/src/auth/index.ts

```diff
@@ -0,0 +1,15 @@
+import Keycloak from "keycloak-js"
+
+export const getToken: () => Promise<string | undefined> = async () => {
```

Comment:
How does this code interact with the server-side endpoint at /keycloak.json? Does it look for server settings at that URL by default?

If so, it would probably by worth a comment in the controller explaining that /keycloak.json is a standard path that the JS depends on.

Reply:
Yep, it's the default location for the settings. I've added a comment to the controller.